dev/MRTCore/mrt/NuGet/Microsoft.ProjectReunion.nuspec

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+     <metadata minClientVersion="3.3">
+         <id>Microsoft.ProjectReunion</id>
+         <version>1.0</version>
+         <title>$id$</title>
+         <authors>Microsoft</authors>
+         <description>Resource loader for .PRI files.</description>
```

Comment:
>Resource loader for .PRI files. [](start = 22, length = 31)

description is not correct now.

Reply:
Fixed, moved file out of this directory and changed description